Automattic/wp-calypso
# Blog onboarding: Create new setup-blog step

Related to https://github.com/Automattic/dotcom-forge/issues/2217

## Proposed Changes

* This PR adds a functional "Setup your blog" task to the "Start writing" Launchpad flow.


https://user-images.githubusercontent.com/140841/235708701-e94a147a-a503-4f5c-8834-ddd02f5a04db.mp4



## Testing Instructions



* Load this PR
* Go to http://calypso.localhost:3000/setup/start-writing/launchpad?siteSlug={your-test-site}&start-writing=true on a fresh test site.
* "Setup your blog" should not be scored and should be clickable
* Click "Setup your blog" and update the name and description of your blog. Click continue.
* You should be taken back to the Launchpad and "Setup your blog" should be scored out.
* Re-enter "Setup your blog" to confirm that your site name and description actually saved.

## Pre-merge Checklist
